server/graphql/v2/interface/Account.js

```diff
@@ -613,13 +612,8 @@ const accountFieldsDefinition = () => ({
     type: new GraphQLNonNull(Policies),
     description:
       'Policies for the account. To see non-public policies you need to be admin and have the scope: "account".',
-    async resolve(account, _, req) {
-      const policies = account.data?.policies || {};
-      if (req.remoteUser?.isAdminOfCollective(account) && checkScope(req, 'account')) {
-        return policies;
-      }
-
-      return pick(policies, PUBLIC_POLICIES);
+    async resolve(account) {
+      return account;
```

Comment:
We need to do a similar change in the v1 api because the same `Policies` v2 object is used there